pillar/salt_master.sls

```diff
@@ -165,6 +166,17 @@ salt_master:
             - salt://reactors/slack/post_event.sls
         - salt/engine/sqs/mitxpro-production-autoscaling:
             - salt://reactors/mitxpro/edxapp_ec2_autoscale.sls
+    engine:
```

Comment:
nitpick: I would pluralize this since we will probably add other engines over time

Reply:
Funny u mention it. I had it pluralized, but was worried that it might fail for some reason due to duplicate keys